website/docs/mlflow/mlflow_on_synapse_analytics.md

```diff
@@ -0,0 +1,161 @@
+# Log Model with MLflow on Synapse Analytics
+
+
+## Prerequisites:
+
+To log model with mlflow on Synapse Analytics for the first time, you will need to 
+
+ - Create an AML workspace
+ - Create a Service Principle with AML
+ - Create a Key Vault to manage secrets (recommended, optional)
+
+
+### Create AML resources
+
+Please follow this document to create [AML workspace](https://learn.microsoft.com/en-us/azure/machine-learning/quickstart-create-resources#create-the-workspace), compute instance and compute clusters are not required.
+
+
+### Create a Service Principle with AML
+
+- Install Azure CLI: You will need to install [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) to create a service principal.
+
+- Find the subscription id: You will need your subscription id to create a service principal, which can be found [here](https://portal.azure.com/#view/Microsoft_Azure_Billing/SubscriptionsBlade)
+
+<img src="https://mmlspark.blob.core.windows.net/graphics/Documentation/subscription_id.png" width="600" style="float: center;"/>
+
+- Create a service principal: Please follow this document to create a [service principal](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-setup-authentication?tabs=sdk#configure-a-service-principal), you will need the subscription id from the step above. Follow the step 1 and 2 under configure a service principal, and take a note of your output JSON. The result will be similar to
+```json
+
+{
+    "clientId": "your-client-id",
+    "clientSecret": "your-client-secret",
+    "subscriptionId": "your-sub-id",
+    "tenantId": "your-tenant-id",
+    "activeDirectoryEndpointUrl": "https://login.microsoftonline.com",
+    "resourceManagerEndpointUrl": "https://management.azure.com",
+    "activeDirectoryGraphResourceId": "https://graph.windows.net",
+    "sqlManagementEndpointUrl": "https://management.core.windows.net:5555",
+    "galleryEndpointUrl": "https://gallery.azure.com/",
+    "managementEndpointUrl": "https://management.core.windows.net"
+}
+
+```
+
+### Install authentication package
+
+The code below install package for this session. If you want to install library on your cluster, please follow this [document](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-use-mlflow-azure-synapse?tabs=cli%2Cmlflow#install-libraries).
+
+
+```python
+import subprocessimport 
+import sys
+subprocess.check_call([sys.executable, "-m", "pip", "install", "azure.ai.ml"])
+```
+
+### Configure authentication
+
+
+```python
+# Please manage the secrets with Azure KeyVault. The following code example is for quick test not for production.
+import os
+os.environ["AZURE_TENANT_ID"] = "<tenantId> in the service principle json above"
```

Comment:
COnsider making this a real notebook as opposed to an untested markdown. You can use the find_secret api to automate the keyvault integration